src/recipes/heartbeat-by-user-first-impression/index.js

```diff
@@ -245,23 +247,22 @@ let run = function (state, extras) {
     }
   };
 
-  let breaks = [1.0];
-
 
+  let channel_breaks = {'aurora':[.3]};
```

Comment:
We only need a few hundred users, and I was thinking of running for a full week to keep weekday/weekend users.  If oversampling isn't an issue then I could just do 1.0.

Also, this doesn't seem to be working in practice.  I seem to be getting the survey 100% on Aurora.